### ISSUE TITLE
Long unbroken text values in list columns override explicit widths

### ISSUE THREAD

**Issue by ayumihamasaki2019** [octobercms/october#4618]
Basically someone has added a PR and merged it into the develop branch and not properly tested it.

Someone has now added this to all tables:

```css
word-break: break-word;
word-wrap: break-word;
```

So now all the words in tables are cut up and not reading correctly and you haven't fixed the basic issue of making a table responsive!

Here is a correct solution for tables in October backend:

```html
<div style="overflow-x:auto;">
  <table>
    ...
  </table>
</div>
```

Then remove the bad css code please:

```css
word-break: normal;
word-wrap: normal;
```

I believe the pr creating this issue is here: https://github.com/octobercms/october/pull/4236

Taking this one step further:

Because people are lazy to add this to their tables, you can inject the HTML wrapper into the dom.

I have already done this to fix my plugins:

```js
$(function () {
    // Wrap the table with a container with css
    $(.form-control table').wrap('<div style="overflow-x:auto;"></div>');
});
```

p.s. If people add css changes can they try to isolate their css code to specific elements and not every single form element, please!

**Comment by bennothommo** [comment]
@w20k Doesn't `table-layout: fixed` just simply lock the table to the specified column widths? I think it's better auto-sizing the columns based on content, which has been the default behaviour so far. Adding an extra class to the table to optionally enable word-wrap might be a good idea, as then the ball is in the court of the developer to decide if it's worth enabling or not.

I also think @ayumihamasaki2019's idea of a scrollable container is a good one, although it needs to be done in the widget templates, not as a JS fix.

**Comment by Samuell1** [comment]
@bennothommo table is already scrollable, if you hold on labels in table

**Comment by ayumihamasaki2019** [comment]
Just tested @Samuell1 comment out, I did get the list widget to scroll, it's not labels its the <th> elements.

@bennothommo 

But it's very confusing and **_not a good design_**. It would be better to change the `mouse cursor` from `pointer `to `grab` icon.

### current setup in list widget

All table hover is `cursor: pointer;`

![1](https://user-images.githubusercontent.com/55155131/64959667-88473d00-d889-11e9-9a88-2c87c0d14f8c.gif)

### changing cursor to grab in list widget

Only table header is `cursor: grab` and rest is `cursor: pointer;`

![2](https://user-images.githubusercontent.com/55155131/64959809-dc522180-d889-11e9-95b5-5742998457ee.gif)

Though, I think a better design could be done...

**Comment by LukeTowers** [comment]
@ayumihamasaki2019 feel free to submit a PR for the grab cursor icon, I like that as a good minor improvement in the meantime.

**Comment by ayumihamasaki2019** [comment]
Done. 👍

**Comment by daftspunk** [comment]
Fixed by #4629
